src/ImageSharp/Formats/Jpeg/Components/Decoder/FastACTables.cs

```diff
@@ -1,90 +1,26 @@
 ﻿// Copyright (c) Six Labors and contributors.
 // Licensed under the Apache License, Version 2.0.
 
-using System;
 using System.Runtime.CompilerServices;
 
-using SixLabors.ImageSharp.Memory;
-using SixLabors.Memory;
-
 namespace SixLabors.ImageSharp.Formats.Jpeg.Components.Decoder
 {
     /// <summary>
     /// The collection of lookup tables used for fast AC entropy scan decoding.
     /// </summary>
-    internal sealed class FastACTables : IDisposable
+    internal sealed class FastACTables
```

Comment:
Maybe this whole class could be replaced with an array now.

Reply:
Agreed. Will do.